doc/sphinx/proof-engine/tactics.rst

```diff
@@ -1896,4 +1880,5 @@ Performance-oriented tactic variants
               native_cast_no_check I.
             Fail Qed.
 
-.. [2] This corresponds to the cut rule of sequent calculus.
+.. tacn:: destauto {? in @ident }
+   :undocumented:
```

Comment:
For some reason, you seem to have added this tactic twice to this chapter.

BTW, this tactic was introduced in f3a53027589813ff19b3a7c46d84e5bd2fc65741. It is not used in the stdlib and was never documented, so it's probably not much used. We should decide if we want to document it or remove it, but it seems useful.